dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/EnrollmentPersister.java

```diff
@@ -100,10 +100,10 @@ protected void updateDataValues(
 
   @Override
   protected void persistComments(TrackerPreheat preheat, Enrollment enrollment) {
-    if (!enrollment.getComments().isEmpty()) {
-      for (TrackedEntityComment comment : enrollment.getComments()) {
+    if (!enrollment.getNotes().isEmpty()) {
+      for (Note comment : enrollment.getNotes()) {
         if (Objects.isNull(preheat.getNote(comment.getUid()))) {
-          this.trackedEntityCommentService.addTrackedEntityComment(comment);
+          this.noteService.addTrackedEntityComment(comment);
```

Comment:
As the only method used in the `NoteService` is this one, could we actually use directly `HibernateSession` here (as we are doing for Tracker Objects) and get completely rid of the `NoteService`?

Reply:
yes good idea